README.md

```diff
@@ -13,9 +13,11 @@ Currently, the official documentation supports two languages:
 
 You can use Google Translate to view the documentation in different languages. For example:
 
-- `fr`: [documentation in French](https://translate.google.com/translate?hl=en&sl=en&tl=fr&u=https%3A%2F%2Fgithub.com%2Fpingcap%2Fdocs%2Fblob%2Fmaster%2FTOC.md)
-- `ja`: [documentation in Japanese](https://translate.google.com/translate?hl=en&sl=en&tl=ja&u=https%3A%2F%2Fgithub.com%2Fpingcap%2Fdocs%2Fblob%2Fmaster%2FTOC.md)
-- `ko`: [documentation in Korean](https://translate.google.com/translate?hl=en&sl=en&tl=ko&u=https%3A%2F%2Fgithub.com%2Fpingcap%2Fdocs%2Fblob%2Fmaster%2FTOC.md)
+- `fr`: [documentation in French](https://docs-pingcap-com.translate.goog/tidb/stable?_x_tr_sl=auto&_x_tr_tl=fr&_x_tr_hl=en&_x_tr_pto=wapp&_x_tr_hist=true)
+- `ja`: [documentation in Japanese](https://docs-pingcap-com.translate.goog/tidb/stable?_x_tr_sl=auto&_x_tr_tl=ja&_x_tr_hl=en&_x_tr_pto=wapp&_x_tr_hist=true)
```

Comment:
None of these links seems to work, and contents remain untranslated. I'm not sure if it's just my network issue. 

<img width="1520" alt="image" src="https://github.com/pingcap/docs/assets/59654584/b02e36f0-5931-4e77-a180-7054743cdd4a">
<img width="1527" alt="image" src="https://github.com/pingcap/docs/assets/59654584/55797f20-a232-45dc-8ada-fe8e2100eea1">

Reply:
What browser are you using? This works for me on Firefox 121.0.1 and Chrome 120.0.6099.224

Reply:
Maybe you can try going to https://translate.google.com and then enter `https://docs.pingcap.com/tidb/stable/overview` in the left box and select something like "Spanish" on the right and then click the URL in the right box?

Reply:
Another option is https://addons.mozilla.org/en-US/firefox/addon/firefox-translations/ but that's browser specific and can't be integrated into the readme as easily.

Reply:
![image](https://github.com/pingcap/docs/assets/1272980/280a3670-6bcf-4da1-9c77-1a2e0285315e)

If you open the network tab of the developer console and filter on `translate.googleapis.com/translate` do you see something like this?

Reply:
Yes. I can see the request fail. It happens when I don't use a global VPN to bypass the GFW. 
<img width="938" alt="image" src="https://github.com/pingcap/docs/assets/59654584/47864c02-a47f-4955-9691-cb4a6008c5e2">

With a VPN it's fine:

<img width="939" alt="image" src="https://github.com/pingcap/docs/assets/59654584/6ac354c6-f2d9-4b50-a4b9-be5b025c8589">